<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frequencyedit</class>
 <widget class="QDialog" name="frequencyedit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>BlankCursor</cursorShape>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Frq</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 87, 83)</string>
  </property>
  <widget class="QPushButton" name="BTN_1">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>110</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">    border: 2px solid grey;
    border-radius: 5px;
    text-align: center;
	color:white
</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_2">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>110</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">    border: 2px solid grey;
    border-radius: 5px;
    text-align: center;
	color:white
</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_3">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>110</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">    border: 2px solid grey;
    border-radius: 5px;
    text-align: center;
	color:white
</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_4">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>110</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">    border: 2px solid grey;
    border-radius: 5px;
    text-align: center;
	color:white
</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_5">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>110</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">    border: 2px solid grey;
    border-radius: 5px;
    text-align: center;
	color:white
</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_6">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>110</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">    border: 2px solid grey;
    border-radius: 5px;
    text-align: center;
	color:white
</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_7">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>180</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">    border: 2px solid grey;
    border-radius: 5px;
    text-align: center;
	color:white
</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_8">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>180</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">    border: 2px solid grey;
    border-radius: 5px;
    text-align: center;
	color:white
</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_9">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>180</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">    border: 2px solid grey;
    border-radius: 5px;
    text-align: center;
	color:white
</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_COLON">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>180</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">    border: 2px solid grey;
    border-radius: 5px;
    text-align: center;
	color:white
</string>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_0">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>180</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">    border: 2px solid grey;
    border-radius: 5px;
    text-align: center;
	color:white
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_BACK">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>180</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">    border: 2px solid grey;
    border-radius: 5px;
    text-align: center;
	color:white
</string>
   </property>
   <property name="text">
    <string>âŒ«</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_OKAY">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(138, 226, 52);</string>
   </property>
   <property name="text">
    <string>Okay</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_ABORT">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>250</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(239, 41, 41);</string>
   </property>
   <property name="text">
    <string>Abort</string>
   </property>
  </widget>
  <widget class="QLabel" name="Frequency">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>32</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(238, 238, 236); color: black;
    border: 2px solid grey;
    border-radius: 5px;
checked{background-color:rgb(238, 238, 236); color: black }
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>106.80 MHz</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/Elektor-Logo.png</normaloff>:/icons/Elektor-Logo.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>160</width>
     <height>96</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>Frequency</zorder>
  <zorder>BTN_1</zorder>
  <zorder>BTN_2</zorder>
  <zorder>BTN_3</zorder>
  <zorder>BTN_4</zorder>
  <zorder>BTN_5</zorder>
  <zorder>BTN_6</zorder>
  <zorder>BTN_7</zorder>
  <zorder>BTN_8</zorder>
  <zorder>BTN_9</zorder>
  <zorder>BTN_COLON</zorder>
  <zorder>BTN_0</zorder>
  <zorder>BTN_BACK</zorder>
  <zorder>BTN_OKAY</zorder>
  <zorder>BTN_ABORT</zorder>
  <zorder>pushButton</zorder>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
